localization/english/religion/wc_religion_l_english.yml

```
@@ -5,4 +5,502 @@
+ 
+ scarlet_light:0 "Scarlet Light"
+ scarlet_light_adj:0 "Scarlet Light"
+ scarlet_light_adherent:0 "Scarlet Follower"
+ scarlet_light_adherent_plural:0 "Scarlet Followers"
+ scarlet_light_desc:0 "A heretical, militant offshoot of the Church of the Holy Light. They are intolerant of all other confessions, regardless of alliance or affiliation. They believe that any and all unholy creatures must be destroyed by a cleansing flame."
+ 
+ naaru:0 "Naarism"
+ naaru_adj:0 "Naaric"
+ naaru_adherent:0 "Naarist"
+ naaru_adherent_plural:0 "Naarists"
+ naaru_desc:0 "The Light manifests itself as a golden, shining light energy. The naaru, enigmatic beings composed of shards of fractured light taught the use of the Light to the draenei and other enlightened races, but others have come into the study and practice of Light-based magic on their own accord."
+ 
+ lightbound:0 "Lightbound"
+ lightbound_adj:0 "Lightbound"
+ lightbound_adherent:0 "Lightbound"
+ lightbound_adherent_plural:0 "Lightbound"
+ lightbound_desc:0 "A heretical offshoot of the naaru teachings, the adherents to the Lightbound doctrine are dedicated to spread the influence of the Light wherever they possibly can and convert the unbelievers by any means necessary, even forcibly."
+ 
+ loa_group:0 "Loa"
+ loa_group_adj:0 "Loa"
+ loa_group_adherent:0 "Loa Follower"
+ loa_group_adherent_plural:0 "Loa Followers"
+ loa_group_desc:0 "Loa is the name given by trolls to beings they worship such as the Wild Gods or spirits. This large group of cults originated in ancient times when the first mortals started encountering magical creatures and deifying them."
+ 
+ zandism:0 "Zan'atali"
+ zandism_adj:0 "Zan'atali"
+ zandism_adherent:0 "Zan'atali"
+ zandism_adherent_plural:0 "Zan'atali"
+ zandism_desc:0 "A pantheon of Loa founded by the Zandalari trolls. This is one of the most ancient pantheons. However, it is renowned for its adaptability. $zandism_adherent_plural$ respect all other Loa in order to maintain friendly ties with other confessions."
+ 
+ gurubism:0 "Gur'atali"
+ gurubism_adj:0 "Gur'atali"
+ gurubism_adherent:0 "Gur'atali"
+ gurubism_adherent_plural:0 "Gur'atali"
+ gurubism_desc:0 "A pantheon of Loa founded by the Gurubashi trolls. The concept of Death occupies a significant place in this pantheon together with Bwonsamdi, Loa of Death. $gurubism_adherent_plural$ pay special attention to interactions with the body of the deceased like ritual cannibalism."
+ 
+ drakkism:0 "Drak'atali"
+ drakkism_adj:0 "Drak'atali"
+ drakkism_adherent:0 "Drak'atali"
+ drakkism_adherent_plural:0 "Drak'atali"
+ drakkism_desc:0 "A pantheon of Loa founded by the Drakkari trolls. Living in harsh conditions requires a strong body and a keen mind. The Drakkari spend their entire lives making stronger and smarter as their Loa demand."
```

Changed the last sentence a little.
```suggestion
 drakkism_desc:0 "A pantheon of Loa founded by the Drakkari trolls. Living in harsh conditions requires a strong body and a keen mind. The Drakkari spend their entire lives striving to become stronger and smarter, as their Loa demand."
```